<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertAlarm</class>
 <widget class="QDialog" name="InsertAlarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>411</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>411</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn_addr">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>45</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_datatype">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>15</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>位</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16位整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16位正整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位正整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16位BCD码</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位BCD码</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位浮点数</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="edit_addr">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>45</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LW0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>44</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>地址</string>
    </property>
    <property name="buddy">
     <cstring>edit_addr</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>15</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据类型 </string>
    </property>
    <property name="buddy">
     <cstring>combo_datatype</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_length">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>44</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>15</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>语言</string>
    </property>
    <property name="buddy">
     <cstring>combo_datatype</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>45</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>连续个数</string>
    </property>
    <property name="buddy">
     <cstring>spin_length</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_language">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>14</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_msg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>报警消息</string>
    </property>
    <property name="buddy">
     <cstring>edit_addr</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>591</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>发送短信</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>该报警产生后将发送报警消息到预设的电话号码，仅支持带3G功能的机型</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>591</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>报警值设置</string>
      </property>
      <widget class="QComboBox" name="combo_Condtion">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>位ON报警</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>位OFF报警</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>报警条件</string>
       </property>
       <property name="buddy">
        <cstring>edit_addr</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_high">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>66</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>上限值</string>
       </property>
       <property name="buddy">
        <cstring>edit_addr</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="edit_low">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>报警值</string>
       </property>
       <property name="buddy">
        <cstring>edit_addr</cstring>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>其他</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="check_screen">
         <property name="text">
          <string>打开画面</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="check_print">
         <property name="text">
          <string>打印</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="check_Time">
         <property name="text">
          <string>打印时间</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="check_add">
         <property name="text">
          <string>添加到数据库</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="check_date">
         <property name="text">
          <string>打印日期</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="combo_screen"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
